package-lock.json

```diff
@@ -1,5485 +1,8 @@
 {
   "name": "football-assets",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
I don't think there should be THIS many changes to the package-lock! Do you know which version of NPM you're using? I think you need to be using atleast NPM v7 or else it will revert to lockfileVersion 1 instead of 2!

I thought `nvm` might install the correct version of npm, but maybe it doesnt...